### PR TITLE
Issue #3183002 by tBKoT: Add tags filtering to the #lazy_builder

### PR DESCRIPTION
**Pull request by dmytriikaiun**
## Problem
New Activity Stream pulls content not defined by selected tag

## Solution
Add tag filter from views to the #lazy_builder

## Issue tracker
https://www.drupal.org/project/social/issues/3183002
https://getopensocial.atlassian.net/browse/YANG-4213

## How to test
*For example*
- [ ] Create a dashboard
- [ ] Add activity stream with tags filter
- [ ] Activity stream shows all activities without filtering

## Screenshots
N/A

## Release notes
N/A

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by ribel** [comment]
@dmytriikaiun Is this change related to this PR? https://github.com/goalgorilla/open_social/pull/1783
if so, since parent PR is not merged yet - you either need to point this PR to parent, or add changes directly in the parent.
Please fix and updated related project patches.

File: modules/social_features/social_activity/social_activity.module [hunk 1]
```diff
@@ -288,6 +288,20 @@ function social_activity_preprocess_block(&$variables) {
           ],
         ],
       ];
+
```
Comment (ribel) [inline modules/social_features/social_activity/social_activity.module:291]: @dmytriikaiun Please add explanation comment here.

**Review by ribel — CHANGES_REQUESTED** [review]
See comments and fix.

**Comment by dmytriikaiun** [comment]
@ribel yes, I will move changes to the parent PR. However, I know only one project that uses this PR.

**Comment by dmytriikaiun** [comment]
Changes moved to #1783

**Pull request closed without merging**